### ISSUE TITLE
Please release Source android 12 for Xiaomi 11T/11T Pro (agate/vili)

### ISSUE THREAD

**Issue by Marvin732** [MiCode/Xiaomi_Kernel_OpenSource#2517]
Can you please release the source for those two devices? Many people would really appreciate it!

**Comment by kotah81** [comment]
Isn't 11T a mediatek powered phone?

**Comment by Marvin732** [comment]
> Isn't 11T a mediatek powered phone?

Yeah it is, devices like the Redmi Note 8 Pro begonia also have their source up to Android R thats why i was wondering about it.

**Comment by Shubhakashyap** [comment]
Release kernel sources please

**Comment by popy2k14** [comment]
+1 @mi-code
